gnuplot_files/plot_memory_base.gp

```diff
@@ -20,8 +20,8 @@ set output "__OUT_FILENAME__"
 set style fill solid border -1
 set grid xtics
 set grid ytics
-set xtics 100000
-set yrange [0:100]
+set xtics 1000000
```

Comment:
Do things work OK if you just remove this? I found that to work well for some of the other continuous monitor scripts.